File: AEDS/C#/Agenda em CSharp/Endereco.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Agenda_em_CSharp
{
    class Endereco
    {

        private int id;
        private string logradouro;
        private int numero;
        private string complemento;
        private string bairro;
        private string cidade;
        private string estado;
        private string CEP;
        private static int contador = 0;

        public Endereco(string Logradouro, int Numero, string Complemento, string Bairro,
                        string Cidade, string Estado, string CEP) {

            this.logradouro = Logradouro;
            this.numero = Numero;
            this.complemento = Complemento;
            this.bairro = Bairro;
            this.cidade = Cidade;
            this.estado = Estado;
            this.CEP = CEP;
            this.id = Endereco.contador;
            Endereco.contador++;
        }

        public int GetId() { return this.id; }

        public string GetLogradouro() { return this.logradouro; }

        public int GetNumero() { return this.numero; }

        public string GetComplemento() { return this.complemento; }

        public string GetBairro() { return this.bairro; }

        public string GetCidade() { return this.cidade; }

        public string GetEstado() { return this.estado; }

        public string GetCEP() { return this.CEP; }

        public void SetLogradouro(string Logradouro) { this.logradouro = Logradouro; }

        public void SetNumero(int Numero) { this.numero = Numero; }

        public void SetComplemento(string Complemento) { this.complemento = Complemento; }

        public void SetBairro(string Bairro) { this.bairro = Bairro; }

        public void SetCidade(string Cidade) { this.cidade = Cidade; }

        public void SetEstado(string Estado) { this.estado = Estado; }

        public void SetCEP(string CEP) { this.CEP = CEP; }
    }
}

File: AEDS/Java/Trabalho Agenda/src/Agenda/Endereco.java
package Agenda;

public class Endereco {

    private int id;
    private String logradouro;
    private int numero;
    private String complemento;
    private String bairro;
    private String cidade;
    private String estado;
    private String CEP;
    private static int Contador = 0;

    public Endereco(String logradouro, int numero, String complemento, String bairro,
                    String cidade, String estado, String CEP){

        this.logradouro = logradouro;
        this.numero = numero;
        this.complemento = complemento;
        this.bairro = bairro;
        this.cidade = cidade;
        this.estado = estado;
        this.CEP = CEP;
        this.id = Endereco.Contador;
        Endereco.Contador++;

    }

    // Metodos get

    public int getId(){ return this.id; }

    public String getLogradouro(){ return this.logradouro; }

    public int getNumero(){ return this.numero; }

    public String getComplemento(){ return this.complemento; }

    public String getBairro(){ return this.bairro; }

    public String getCidade(){ return  this.cidade; }

    public String getEstado(){ return this.estado; }

    public String getCEP(){ return this.CEP; }

    // Metodos set

    public void setLogradouro(String logradouro) { this.logradouro = logradouro; }

    public void setNumero(int numero) { this.numero = numero; }

    public void setComplemento(String complemento) { this.complemento = complemento; }

    public void setBairro(String bairro) { this.bairro = bairro; }

    public void setCidade(String cidade) { this.cidade = cidade; }

    public void setEstado(String estado) { this.estado = estado; }

    public void setCEP(String CEP) { this.CEP = CEP; }
}
